### DISCUSSION TITLE
Can't set CRON, and App/app are undefined?

### DISCUSSION THREAD

**Discussion by edge-rps** [deta/cloud-docs#402] [General] [answered]
So I was setting up cron for the first time when I noticed that the app/App export don't exist in the deta npm package, resulting in the inability to create cron jobs.

_This is the console.log of **require('deta')**;_
![image](https://user-images.githubusercontent.com/44068036/185805593-0aeb6253-58c5-44a4-ae4b-d314b871f7f3.png)

**Comment by lemonyte** [comment, marked as answer]
Not sure about the npm package, but it doesn't exist in the Python package either. You're supposed to just pretend its there and it will be available in a Micro environment. As for testing locally, just comment the cron-related lines I guess.

**Reply by lemonyte** [reply]
If I remember correctly, mustafa mentioned in one of the issues that they are planning to remove App and replace it with a different implementation for cron tasks.
